src/warnet/warnet.py

```diff
@@ -55,6 +55,39 @@ def __str__(self) -> str:
             f"Tanks:\n{tanks_str}"
         )
 
+    def _wn_repr(self) -> dict:
```

Comment:
what does "_wn_repr" mean?

Reply:
@willcl-ark

Reply:
Warnet Representation

Its a python thing: https://docs.python.org/3/library/functions.html#repr

But I can change if its blocking?

Reply:
Actually, now that it returns a `dict` I don't think it's a "real" repr :P

In a mid-edit version I had this returning the `tabulate` tables as strings, but no longer...

Reply:
Updated in 807dab37497b535470c16dd50c3574cacc28b097